package.json

```diff
@@ -96,6 +96,7 @@
     "@babel/preset-env": "7.9.6",
     "@babel/preset-react": "7.9.4",
     "@babel/preset-typescript": "7.9.0",
+    "@golevelup/nestjs-testing": "^0.1.2",
```

Comment:
It's a nice testing utility for our project. Especially for mocking context. Here's a snippet from their docs

> This function will create a mock object for you with all sub properties mocked as jest.fn() unless otherwise provided, to allow for easy mocking later on, but more on that later.

I also use this repo as a reference for nestjs testing and that's where I found this utility.  [testing-nestjs](https://github.com/jmcdo29/testing-nestjs)